Makefile

```diff
@@ -13,19 +12,18 @@ _LDFLAGS += -s
 endif
 
 EDK2OBJDIR = backends/edk2-compat
-_EDK2_OBJ =  edk2-svc-read.o edk2-svc-write.o edk2-svc-validate.o edk2-svc-verify.o edk2-svc-generate.o
+_EDK2_OBJ =   edk2-svc-validate.o edk2-svc-read.o
+#_EDK2_OBJ =  edk2-svc-read.o edk2-svc-write.o  edk2-svc-verify.o edk2-svc-generate.o
```

Comment:
If you're not compiling these .c files into these .o files, it shouldn't be necessary to also change the .c files. Smaller diffs will make reviewing the patches much simpler.

Reply:
In response to this and your other comment, I have rebased so that these work-in-progress commits all still compile normally for mbedtls but not with openssl. When I was still porting openssl, it was either to remove/comment out all mbedtls so that I could compile and test the result. Now that I know that porting is successful, it makes sense that I changed these commits so they are backwards compatible.